packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -77,7 +77,7 @@ const StyledItem = styled('button', {
 
   padding-bottom: ${({ theme }) => theme.spacing(1)};
   padding-left: ${({ theme }) => theme.spacing(1)};
-  padding-right: ${({ theme }) => theme.spacing(1)};
+  padding-right: ${({ theme }) => theme.spacing(0.5)};
```

Comment:
@ehconitin this will impact all navigation drawer items, is this expected? (not only the workspace favorites). This seems suspicious!

Reply:
<img width="984" alt="Screenshot 2024-12-07 at 10 35 24 PM" src="https://github.com/user-attachments/assets/30c0baeb-1ced-4351-b365-751ec18782f7">

@charlesBochet  :)

Reply:
Ok but how is it now broken on main branch ? it seems fine to me at least on production?

Reply:
I am not sure! I went through history of NavigationDrawerItem and it was always like this i.e, 4px padding all around

Reply:
@charlesBochet should we close this? or do you need me to dig deeper into it?